src/setup.h

```diff
@@ -15,7 +17,7 @@ void
 uzbl_events_free ();
 
 void
-uzbl_gui_init ();
+uzbl_gui_init (WebKitWebContext *context);
```

Comment:
Considering if it would be better to store the context on the main struct. Maybe nested under a UzblWeb struct? WDYT @mathstuf ?